monai/handlers/checkpoint_loader.py

```diff
@@ -21,23 +22,34 @@ class CheckpointLoader:
     And also can restore training if load the state_dict of Ignite engine.
 
     Args:
-        load_path (string): the file path of checkpoint, it should be a PyTorch pth file.
+        load_path (str): the file path of checkpoint, it should be a PyTorch pth file.
         load_dict (dict): target objects that load checkpoint to. examples::
 
             {'network': net, 'optimizer': optimizer, 'engine', engine}
 
+        name (str): identifier of logging.logger to use, if None, defaulting to ``engine.logger``.
+        convert_to_multi_gpu (bool): if the checkpoint was trained by single GPU and
+            want to load to multi GPU model, need to load to `object.module` instead.
+
     """
 
-    def __init__(self, load_path, load_dict):
+    def __init__(self, load_path, load_dict, name=None, convert_to_multi_gpu=False):
         assert load_path is not None, "must provide clear path to load checkpoint."
         self.load_path = load_path
         assert load_dict is not None and len(load_dict) > 0, "must provide target objects to load."
+        self.logger = None if name is None else logging.getLogger(name)
+        if convert_to_multi_gpu:
+            for k, v in load_dict.items():
+                if hasattr(v, "module"):
+                    load_dict[k] = v.module
```

Comment:
nice update, do we need an option for `convert_to_multi_gpu`? Iooks like it should always be True, and it won't break the existing single gpu functionality

Reply:
Same as CheckpointSaver, I will delete this option directly.
Thanks.

Reply:
deleted in the latest patch.
Thanks.